### ISSUE TITLE
City of Stirling Location

### ISSUE THREAD

**Issue by ohare93** [HartBlanc/anki-uk-geography#12]
Minor-est of low priority complaints, but hear me out :sweat_smile: 

Why is Stirling almost directly above Glasgow? It should be slightly more to the right.

![image](https://user-images.githubusercontent.com/1471350/148332834-79ddfcd6-dfd3-483f-904d-56041c54d1f0.png)

Let's compare it on OpenStreetMaps and Google Maps
![image](https://user-images.githubusercontent.com/1471350/148332889-a0bf4164-e3ad-4d15-8837-ca73cbb9f3e0.png)
![image](https://user-images.githubusercontent.com/1471350/148332956-d306613e-5e3b-4035-9648-4536e3fa5de8.png)

Out of curiosity, where did you get the city coordinates from?

**Comment by HartBlanc** [comment]
Hey - thanks for pointing this out. I've updated the deck now as suggested.

The positions I picked for the cities is based on the map on this Wikipedia page:
https://en.wikipedia.org/wiki/List_of_cities_in_the_United_Kingdom

The actual position is determined by my manual approximation of mapping that onto my own map so I must have just misjudged the distances here. I'd like to be a bit more precise (e.g. base position in map on lat/long) and may look into re-working this in the future.

Thanks again!
